build.gradle

```diff
@@ -468,6 +485,9 @@ subprojects {
             if (child.bundle != null)
                 bundle = child.bundle.dest
             CONFIG?.libraries?.get(side)?.each { library "'${it}'" }
+            if (MERGE_PATCHES && (side == 'client' || side == 'server' || side == 'joined')) {
+                library "project(':${project.version.toString()}-shared')"
```

Comment:
In my local testing, this dependency fails to resolve because the `-shared` project does not expose any variant (because it's missing a `build.gradle` file with the right dependencies).